src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -17,25 +17,25 @@ public static class TreeExtensions
         /// Predict a target using a decision tree regression model trained with the <see cref="FastTreeRegressionTrainer"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="RegressionCatalog"/>.</param>
-        /// <param name="labelColumn">The label column.</param>
-        /// <param name="featureColumn">The feature column.</param>
-        /// <param name="weights">The optional weights column.</param>
+        /// <param name="labelColumn">The name of the label column.</param>
+        /// <param name="featureColumn">The name of the feature column.</param>
+        /// <param name="weights">The name of the optional weights column.</param>
```

Comment:
>weights [](start = 25, length = 7)

can you update this one as well, otherwise build is not working

Reply:
Sorry I meant in whole file. We have strict rules to match parameters names in xml documentation

---
In reply to: [255319474](https://github.com/dotnet/machinelearning/pull/2432#discussion_r255319474) [](ancestors = 255319474)

Reply:
Should be ok now